Doc/library/sys.rst

```diff
@@ -1060,6 +1062,31 @@ always available.
         This is still called as a fallback if a :data:`meta_path` entry doesn't
         have a :meth:`~importlib.abc.MetaPathFinder.find_spec` method.
 
+.. data:: module_names
+
+   A tuple of strings giving the names of standard library modules.
+
+   The list is the same on all platforms. Modules which are not available on
+   some platforms and modules disabled at Python build are also listed.
+   All module kinds are listed: pure Python, built-in, frozen and extension
+   modules.
+
+   For packages, only sub-packages are listed, not sub-modules. For example,
+   ``concurrent`` package and ``concurrent.futures`` sub-package are listed,
+   but not ``concurrent.futures.base`` sub-module.
+
+   Some special stdlib modules are excluded, like test and private modules.
+
+   See also the :attr:`sys.builtin_module_names` list.
+
+   .. note::
+      If a third party module has the same name than a standard library module
```

Comment:
Why this note? How is this information relevant to this API? The name itself is the same, right? Or what we want to say is that there may be 3rd party modules with the same name as stdlib modules?

Reply:
If you put an "asyncio.py" in the current directory, run Python and type "import asyncio", you get your module instead of the stdlib module. That's what I mean by "same name".

It's a note related to https://bugs.python.org/issue42955#msg385255 remark.

Reply:
I see, but how that affects this list? We should maybe rephrase saying "there is no guarantee that an imported module with a name in this collection is actually an stdlib module as any 3rd party module can iterpose itself by ...."